features/helpers/verification_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module VerificationHelper
+  include RSpec::Matchers
+
+    def verify_object_equals(actual, expected, attribute = nil)
+      expect(actual).to eql(expected), "Data mismatch #{attribute} should be equal to #{expected} but it is #{actual}"
+    end
+
+    def verify_inclusion(actual, expected)
```

Comment:
better to name it more clear, for example:
```suggestion
    def verify_array_include(array, to_include)
```